docs/source/develop.rst

```diff
@@ -35,9 +35,10 @@ non-exhaustive list follows:
 *  http://github.com/dask/dask: The main code repository holding parallel
    algorithms, the single-machine scheduler, and most documentation.
 *  http://github.com/dask/distributed: The distributed memory scheduler
-*  http://github.com/dask/hdfs3: Hadoop Filesystem interface
+*  http://github.com/dask/dask-ml: Hachine learning algorithms
 *  http://github.com/dask/s3fs: S3 Filesystem interface
-*  http://github.com/dask/dask-ec2: AWS launching
```

Comment:
Are you wanting to include `daskernetes` or `dask-kubernetes` instead? Not sure which one is more appropriate as I haven't been following them that closely, but it seems like one (or both) of them would be good to add here.

Reply:
I think we either need to list all of them or none of them here, otherwise we give the overly-narrow impression that kubernetes is the only code out there to launch Dask.  I think that if people see this now they'll think "there must be ways to launch dask on clusters, I'll go check out the setup page or the github.com/dask org".  I think that if we list only daskernetes then people might think "Dask deploys itself with kubernetes.  That's a shame. I use SLURM.  Oh well.

In general I'm afraid of making precise and incomplete lists.  This probably carries over to my thoughts on the FAQ as well.

Reply:
Sure. That makes sense. Though I suppose the same could be said of the FS resources (e.g. we don't mention `dask-adlfs`).

Maybe these things could live on some other page people could drill down into. Would suggest a cloud page or perhaps there is an existing page that already works well for this.

Reply:
@martindurant what is the current status of adlfs?  Ok to link to?